### PR TITLE
#5988 - feat: styling footer.

### PR DESCRIPTION
**Pull request by DipakShrestha18**
@chriddell @creido-welly This PR is not complete yet but I though this will good starting point if you want to look at it.

### CODE REVIEW

File: src/components/Footer/_footer.scss [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+.footer-nav__list--grid {
+  @include mq(sm) {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0 -10px;
+
+    > .footer-nav__item {
+      margin: 0 10px;
+      width: calc(25% - #{10px});
```
Comment (creido-welly) [inline src/components/Footer/_footer.scss:82]: should probably be  `width: calc(25% - #{20px});` to counter the horizontal margins on left AND right

**Review by creido-welly — APPROVED** [review]
just one item to double check

**Comment by chriddell** [comment]
Cheers @DipakShrestha18 - I'm gonna pick the rest of this up today.

**Comment by DipakShrestha18** [comment]
Thanks @chriddell

**Comment by chriddell** [comment]
Closing this in favour of https://github.com/wellcometrust/corporate-components/pull/95.

@DipakShrestha18 I built on your initial work and refactored to slim down the styles as much as possible.

Have created two more tickets to get this work finished:

https://github.com/wellcometrust/corporate/issues/6007
https://github.com/wellcometrust/corporate/issues/6008

**Pull request closed without merging**